packages/pn-personafisica-webapp/src/components/Contacts/SmsContactItem.tsx

```diff
@@ -304,6 +305,13 @@ const SmsContactItem: React.FC<SmsItemProps> = ({
   const hasDigitalDomicile = !!defaultSERCQ_SENDAddress || !!defaultPECAddress;
 
   const deleteConfirmHandler = () => {
+    if (!fromSercqSend) {
```

Comment:
Why this event is emitted only if sercq is disabled?
On airtable I have this description:

> on click >
> - se domicilio digitale attivo: on click > click su “Disattiva SMS” nella modale di conferma di rimozione del contatto
> - se domicilio digitale non attivo: on click > click su “Conferma” nella modale di conferma di rimozione del contatto